SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -824,6 +824,12 @@ void USpatialNetDriver::NotifyActorDestroyed(AActor* ThisActor, bool IsSeamlessT
 
 	const bool bIsServer = ServerConnection == nullptr;
 
+	// Remove the record of destroyed singletons.
+	if (ThisActor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton))
```

Comment:
If this is happening on Servers, then I'd imagine something's gone wrong. Might be worth a log.

Reply:
Log added. No big deal. Definitely didn't take all of Friday to add.